src/sentry/models/organization.py

```diff
@@ -310,10 +359,22 @@ def default_owner_id(self):
     def has_single_owner(self):
         from sentry.models import OrganizationMember
 
-        count = OrganizationMember.objects.filter(
-            organization=self, role=roles.get_top_dog().id, user__isnull=False, user__is_active=True
-        )[:2].count()
-        return count == 1
+        owners = list(
+            OrganizationMember.objects.filter(
+                organization=self,
+                role=roles.get_top_dog().id,
+                user__isnull=False,
+                user__is_active=True,
+            ).values_list("id", flat=True)
+        )
+        owners += list(
+            OrganizationMemberTeam.objects.filter(
+                team__in=self.get_teams_with_org_role(organization_roles.get_top_dog().id)
+            )
+            .exclude(organizationmember_id__in=owners)
+            .values_list("id", flat=True)
+        )
+        return len(owners[:2]) == 1
```

Comment:
I don't quite understand this line of code. Why doesn't `len(owners) == 1` work?

Reply:
good point. i'm not sure why the original code has `[:2].count()` and then checks if `count == 1` 
 when `len(owners) == 1` does the same thing

Reply:
it forces a LIMIT on the query I'd assume, though I'm not sure what the ORM does..

you basically want to say "i only need you to scan for two results, and if there are two results, return False"